tests/unit/utils/test_thumbnail_manager.py

```diff
@@ -24,6 +25,28 @@ def manager(qtbot, tmp_path, mocker):
     yield thumbnail_manager.ThumbnailManager(None)
 
 
+def test_thumbnail_save_disabled(monkeypatch, qtbot, tmp_path, manager):
+    monkeypatch.setattr(settings.thumbnail.save, "value", False)
+    no_thumbnail_path = str(tmp_path / "no_thumbnail.jpg")
+    QPixmap(300, 300).save(no_thumbnail_path, "jpg")
+    manager.create_thumbnails_async([no_thumbnail_path])
+    check_thumbails_created(qtbot, manager, 0)
+
+
+def test_thumbnail_save_disabled_no_delete_old(monkeypatch, qtbot, tmp_path, manager):
+    monkeypatch.setattr(settings.thumbnail.save, "value", True)
+    has_thumbnail_path = str(tmp_path / "has_thumbnail.jpg")
+    QPixmap(300, 300).save(has_thumbnail_path, "jpg")
+    manager.create_thumbnails_async([has_thumbnail_path])
+    check_thumbails_created(qtbot, manager, 1)
+
+    monkeypatch.setattr(settings.thumbnail.save, "value", False)
+    no_thumbnail_path = str(tmp_path / "no_thumbnail.jpg")
+    QPixmap(300, 300).save(no_thumbnail_path, "jpg")
+    manager.create_thumbnails_async([has_thumbnail_path, no_thumbnail_path])
+    check_thumbails_created(qtbot, manager, 1)
```

Comment:
Could you elaborate why you believe there is a typo involved?

The function does not start with `test_` as it is not a standalone unit-test, but instead used by various tests to perform the same assert + wait logic.

Reply:
Oh, if you see the function `check_thumbails_created`, thumbails is missing the n

Reply:
Ah, damnit, must have been blind twice, thanks :laughing: